src/Symfony/Component/Security/Core/Authentication/AuthenticationTrustResolverInterface.php

```diff
@@ -20,16 +20,6 @@
  */
 interface AuthenticationTrustResolverInterface
 {
-    /**
-     * Resolves whether the passed token implementation is authenticated
-     * anonymously.
-     *
-     * If null is passed, the method must return false.
-     *
-     * @return bool
-     */
-    public function isAnonymous(TokenInterface $token = null);
```

Comment:
to be deprecated on 5.4 I guess. Shall we suggest `!isFullFledged()` as a replacement, or should we introduce something new?

Reply:
See #42510